app/views/unattended/report_templates/jobs_-_invocation_report_template.erb

```diff
@@ -0,0 +1,42 @@
+<%#
+name: Jobs - Invocation report template
+snippet: false
+template_inputs:
+- name: job_id
+  required: true
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+  hidden_value: false
+- name: hosts
+  required: false
+  input_type: user
+  description: Field for filter hosts of job invocation. Leave blank for all hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+require:
+- plugin: foreman_remote_execution
+￼ version: 4.2.2
+-%>
+<%- invocation = find_job_invocation_by_id(input('job_id')) -%>
+<%- input_hosts = input('hosts') -%>
+<%- input_hosts = '*' if input_hosts.blank? -%>
+<%- load_hosts(search: "#{input_hosts} and job_invocation.id = #{input('job_id')}").each do |batch| -%>
+<%-   batch.each do |host|  -%>
+<%-     task = invocation.sub_task_for_host(host) -%>
+<%-     task.action_output.each do |output| -%>
+<%-       report_row(
+            host: host.name,
+            type: output['output_type'],
+            message: output['output'],
+            ended_at: task.ended_at,
```

Comment:
Shouldn't this also contain outputs' timestamps?

Reply:
Yeah, maybe it would be more useful then `ended_at` on every line :thinking:
User can kinda guess what time it ended by the last line and it's not like it would be so important information.

Reply:
And I think in that case I should have to update PR to Tasks because I f I remember correctly, the `timestamp` is in float. So for proper display I should covert it to `datetime` and update the macro.

Reply:
Aren't we supporting Time.at(double)?

Reply:
Well, I tried call `Time...` in report template and it's not working.

Should I update the macro in tasks? @ezr-ondrej

Reply:
What about adding simple helper method in `lib/foreman/renderer/scope/macros/helpers.rb`?
Something like `def time_to_str(time)` that would take `num/Time` as argument and outputed formatted time?

Reply:
@ezr-ondrej Good idea! I had something similar in mind. I'll do it.